### DISCUSSION TITLE
Switching out components dynamically - is it supposed to work?

### DISCUSSION THREAD

**Discussion by pixelkritzel** [solidjs/solid#992] [Q&A] [answered]
This one of the biggest plus points for me of React and I'm curious if this is a bug or conceptually impossible.

Simple demo here. 

https://codesandbox.io/s/6mjh5j

**Comment by yw662** [comment, marked as answer]
so, isn't it a `Dynamic` ?

**Reply by pixelkritzel** [reply]
Thank you!

For reference: I should have used this.

https://www.solidjs.com/docs/latest/api#%3Cdynamic%3E
